test/unit/LivepeerETHTicketBroker.js

```diff
@@ -181,6 +181,107 @@ contract("LivepeerETHTicketBroker", accounts => {
         })
     })
 
+    describe("fundAndApproveSigners", () => {
```

Comment:
should we add tests that ensure events are emitted?

Reply:
There are already tests for events emitted from the modifiers and `approveSigners` - since this function just internally calls those modifiers and `approveSigners` I did not include tests for events emitted in the interest of avoiding additional duplication. WDYT?

Reply:
Well, you did add tests for tracking balances, which is also logic that lives inside the modifiers :) so we're testing one part of the modifiers' behavior but not the other.

I share the sentiment that this is redundant, but find it harder to let go of the inconsistency.

Perhaps we should consider dropping the test for balance tracking?
Or re-discuss how we could design these contracts to achieve more straightforward code reuse that doesn't force us to duplicate a bunch of testing code?